doc/src/explanation/active-deprecations.md

```diff
@@ -160,15 +160,15 @@ motion.
 >>> from sympy.physics.mechanics import (
 ...   Body, JointsMethod, PinJoint, PrismaticJoint)
 >>> g, l = symbols("g l")
->>> wall = Body("wall")
->>> cart = Body("cart")
->>> pendulum = Body("Pendulum")
->>> slider = PrismaticJoint("s", wall, cart, joint_axis=wall.x)
+>>> wall = Body("wall") # doctest: +SKIP
+>>> cart = Body("cart") # doctest: +SKIP
+>>> pendulum = Body("Pendulum") # doctest: +SKIP
+>>> slider = PrismaticJoint("s", wall, cart, joint_axis=wall.x) # doctest: +SKIP
```

Comment:
@moorepants @tjstienstra I think it is probably worth reviewing exactly what the docs say about what is deprecated etc given that SymPy 1.14 will be released soon.

It looks like there are notes about things that are deprecated and the suggested alternative is using other things that are also then described as deprecated in later sympy versions.

Reply:
Thanks for notifying. We indeed deprecated some kwargs of `Joint` in 1.12 and use `Body`, deprecated in 1.13, as example in this document. Do we want to replace the example to use `RigidBody`? Disadvantage is `RigidBody`'s usage in this aspect is not yet possible in 1.12.

Another minor fix we should do is remove a few `Body` references from class docstrings of joints.

Reply:
Just opened #27950 to update it accordingly.